src/essentials/Colors/docs/SemanticColorsTable.tsx

```diff
@@ -58,23 +105,40 @@ export const CssVariablesTable: FC<{ tier: 'b' | 's' }> = ({ tier }) => {
                             />
                         </Box>
                     </TableHeaderCell>
-                    <TableHeaderCell>Hex Code</TableHeaderCell>
                 </TableRow>
             </thead>
             <tbody>
-                {filteredTokens.map(({ variable, value }) => (
-                    <TableRow key={variable}>
-                        <TableCell>
-                            <ColorBlock token={applyPrefix(variable, tier)} />
-                        </TableCell>
-                        <TableCell>
-                            <code>{variable}</code>
-                        </TableCell>
-                        <TableCell>
-                            <code>{value}</code>
-                        </TableCell>
-                    </TableRow>
-                ))}
+                {filteredTokens.map(({ variable, value, invertedValue }) => {
+                    const token = applyPrefix(variable, tier);
+
+                    return (
+                        <TableRow key={variable}>
+                            <TableCell>
+                                <BlockContainer>
+                                    <ColorBlock token={token} />
+                                </BlockContainer>
+                                <code>{value}</code>
+                            </TableCell>
+                            <TableCell>
+                                <BlockContainer>
+                                    {isDark ? (
```

Comment:
I tried using `InvertedColorScheme` but the color would always be the same, I don't understand why, will try debugging this more tomorrow

Reply:
It is expected, the `InvertedColorScheme` relies on your OS preference, not on Storybook toggle